openfecwebapp/templates/partials/candidate/other-spending-tab.html

```diff
@@ -1,83 +1,84 @@
 {% import 'macros/cycle-select.html' as select %}
 
 <section id="section-3" role="tabpanel" aria-hidden="true" aria-labelledby="section-3-heading">
+  <div class="heading--section">
+    <h2 id="section-3-heading">Spending by others to oppose/support</h2>
+  </div>
   <div class="content__section">
-    <div class="heading--section">
-      <h2 id="section-3-heading">Spending by others to oppose/support</h2>
-    </div>
+    <p class="usa-width-two-thirds">This tab shows spending that opposes or supports this candidate. None of the funds are directly given to or spent by the candidate.</p>
+  </div>
 
+  <div class="slab slab--inline slab--neutral u-padding--left u-padding--right">
```

Comment:
The markup changes here are just to add styles to match the other tabs (with the white boxes on gray backgrounds).